cli/tests/test_auth.rs

```diff
@@ -572,17 +573,58 @@ fn test_cmd_auth_status_env() {
 
 #[test]
 fn test_cmd_auth_debug_logging() {
-    let bad_url = "sys.oxide.invalid";
+    let server = MockServer::start();
 
-    // Validate debug logs are printed
-    Command::cargo_bin("oxide")
+    let oxide_mock = server.current_user_view(|when, then| {
+        when.into_inner()
+            .header("authorization", "Bearer oxide-token-good");
+
+        then.ok(&oxide::types::CurrentUser {
+            display_name: "privileged".to_string(),
+            id: "001de000-05e4-4000-8000-000000004007".parse().unwrap(),
+            silo_id: "d1bb398f-872c-438c-a4c6-2211e2042526".parse().unwrap(),
+            silo_name: "funky-town".parse().unwrap(),
+        });
+    });
+
+    let cmd = Command::cargo_bin("oxide")
         .unwrap()
-        .arg("--debug")
```

Comment:
we're not using --debug becuase it gets other stuff? Do we want to make --debug equivalent to RUST_LOG=oxide=debug?